Assets/MixedRealityToolkit/Inspectors/MixedRealityToolkitInspector.cs

```diff
@@ -50,64 +45,14 @@ public override void OnInspectorGUI()
             }
 
             serializedObject.Update();
-            EditorGUI.BeginChangeCheck();
-            EditorGUILayout.PropertyField(activeProfile);
-            bool changed = EditorGUI.EndChangeCheck();
-            string commandName = Event.current.commandName;
 
             // If not profile is assigned, then warn user
             if (activeProfile.objectReferenceValue == null)
             {
                 EditorGUILayout.HelpBox("MixedRealityToolkit cannot initialize unless an Active Profile is assigned!", MessageType.Error);
-
-                if (GUILayout.Button("Assign MixedRealityToolkit Profile") || forceShowProfilePicker)
```

Comment:
Why is this deleted?

Reply:
The profile picker is no longer necessary now that we have dropdown menus.